code/presentation/mb-client/src/screens/home/find-masjid.tsx

```diff
@@ -0,0 +1,234 @@
+import { useNavigation } from '@react-navigation/native';
+import * as Location from 'expo-location';
+import React, { useState } from 'react';
+import {
+  Button,
+  FlatList,
```

Comment:
We should be using flashlist lib instead

Reply:
Done